index.js

```diff
@@ -111,14 +111,24 @@ function wkhtmltopdf(input, options, callback) {
     var child = spawn(wkhtmltopdf.shell, ['-c', 'set -o pipefail ; ' + args.join(' ') + ' | cat'], spawnOptions);
   }
 
+  var timeout
+  if (options.timeout) {
+    timeout = setTimeout( function () {
```

Comment:
nit - spacing

Reply:
Do you think we can proceed with the merge 🙂 ?